fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonUtils.xtend

```diff
@@ -263,6 +264,16 @@ class PythonUtils {
             )
     }
     
+    def enumLabelsWithNumber(Attribute attr){
+    	var enumVal =0
```

Comment:
Add a space between `=` and `0` would be more consistent with the other code.

Reply:
Done.